client/src/lib/dependency-injection/applyFormMiddleware.js

```diff
@@ -1,7 +1,6 @@
 import Injector from '../Injector';
 import getIn from 'redux-form/lib/structure/plain/getIn';
 import setIn from 'redux-form/lib/structure/plain/setIn';
-import SchemaStateManager from './SchemaStateManager';
 
 const applyFormMiddleware = (reducer) => () => (state, action) => {
```

Comment:
Yeah, welcome to redux. Every reducer, every action. I've added this filter as a black list, in case there are more actions we want to filter out. 

Relevant: https://github.com/omnidan/redux-ignore